packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -324,7 +340,21 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
       if (this.call?.id) {
         this.context.setCurrentCallId(this.call.id);
       }
+
       this.context.updateClientState(clientState);
+
+      /* @conditional-compile-remove(call-transfer) */
+      const _callAgent = callAgent as CallAgent;
+      /* @conditional-compile-remove(call-transfer) */
+      const transferCall = _callAgent.calls.find(
+        (call) =>
+          (call as Call).state === 'Connected' && call.id === this.context.getState().acceptedTransferCallState?.id
+      );
+      /* @conditional-compile-remove(call-transfer) */
+      if (this.call && transferCall && this.call.id !== transferCall.id && transferCall.state === 'Connected') {
```

Comment:
Is it possible to land in another state that isn't `Connected` when being transferred, e.g. lobby or hold?